code/debugconsole/console.cpp

```diff
@@ -488,6 +488,21 @@ void dc_putc(char c)
 	line_str->push_back(c);
 }
 
+bool handle_textInputEvent(const SDL_Event& event) {
+	if (event.text.text[0] != '\0' && event.text.text[0] != '\1') {
+		for (char c : event.text.text) {
+			if (c < 32)
+				break;
+
+			if (dc_command_buf.size() < MAX_CLI_LEN) {
+				dc_command_buf.push_back(c);
+			}
+		}
+	}
+
+	return true;
```

Comment:
This doesn't seem like it can return false.  Why does it return a bool in the first place?

Reply:
This is a handler method that must be of a form that can be accepted by os::events::addEventListener() in osapi, which requires the boolean return value AFAIK.